cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -461,6 +474,7 @@ datasource "ds_azure_instance_size_map" do
 end
```

Comment:
regarding datasource `ds_azure_instance_size_map` (line 467)

The path should be `/flexera-public/policy_templates/master/data/azure/instance_types.json` rather than referencing rightscale